doc/man3/OSSL_PARAM.pod

```diff
@@ -274,7 +257,7 @@ could fill in the parameters like this:
 
     for (i = 0; params[i].key != NULL; i++) {
         if (strcmp(params[i].key, "foo") == 0) {
-            *(char **)params[i].data = "foo value";
+            params[i].data = "foo value";
```

Comment:
Richard: are you trying to say you think that the old form
   *(char **)params[i].data = "foo value";
Is not going to be confusing?

Reply:
Not if you understand that the data passed is the address of the string.
But perhaps that's simply too hard to grasp, so I start to wonder if the `_PTR` types shouldn't simply be dropped. I don't expect them to be used very frequently anyway, so using a normal string and getting a handful of bytes copied isn't a very hard hit...

Reply:
I'm fine dropping the `_PTR` types.  They are only barely used at present.

Reply:
They are used in the get_params, but the values that are being pushed into the PTR params, don't appear to be read anywhere.